scripts/release_pipeline_to_github.sh

```diff
@@ -8,6 +8,7 @@
 # as an environment variable
 set -e
 
+declare -r REPO="broadinstitute/warp"
```

Comment:
Might not be worth it, but there is probably a way to derive this using git commands instead of hardcoding it.

Reply:
Looks like the following command will do it: 
`REPO=$(git remote -v | grep "fetch" | sed 's/git@github.com://; s/origin//; s/.git//; s/(fetch)//')`
Do we like that better?  (I tried to reduce the sed commands, but could not get it to work properly)

Reply:
I think that's better. I got it to work with a slightly simpler `sed:
```
REPO=$(git remote -v | grep fetch | sed -E 's/.*git@github.com:(.*).git.*/\1/')
```

Reply:
Thanks!